ch_pipeline/processing/base.py

```diff
@@ -361,6 +361,55 @@ def pending(self):
 
         return pending
 
+    def completed(self):
```

Comment:
One thing about this is that all the `FINISHED` jobs should have been moved out of the working directory by this point, and to be honest, even if a job has FINISHED in its STATUS file, if it hasn't been moved out of the working directory something went wrong and it should be considered as if it crashed.

Reply:
Resolved